### PR TITLE
[AIRFLOW-4827] Remove compatible test for python 2

### PR DESCRIPTION
**Pull request by zhongjiajie**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-4827

### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

**Comment by OmerJog** [comment]
Shouldnt you target https://issues.apache.org/jira/browse/AIRFLOW-4827 ?

**Comment by zhongjiajie** [comment]
😭 What the matter today. Two PR connect to wrong Jira tickets
Thanks for you remind @OmerJog

File: tests/hooks/test_hive_hook.py [hunk 1]
```diff
@@ -460,23 +458,7 @@ def test_get_results_data(self):
...
-                        delimiter=',', lineterminator='\n', output_header=True, fetch_size=2)
-            df = pd.read_csv(csv_filepath, sep=',')
-            self.assertListEqual(df.columns.tolist(), self.columns)
-            self.assertListEqual(df[self.columns[0]].values.tolist(), [1, 2])
-            self.assertEqual(len(df), 2)
-            self.assertIn('INFO:airflow.hooks.hive_hooks.HiveServer2Hook:'
-                          'Written 2 rows so far.', cm.output)
-
-    @unittest.skipIf(NOT_ASSERTLOGS_VERSION >= 3.4, 'test could cover by test_to_csv_assertLogs')
-    def test_to_csv_without_assertlogs(self):
```
Comment (XD-DENG) [inline tests/hooks/test_hive_hook.py:479]: Not sure if you have removed wrong test case?
The test case you removed will be skipped if Py version < 3.4, i.e., this test case is the one for Py 3.4+ which should be kept.
(**I haven't looked into the test case details. Justing checking the `@unittest.skipIf` conditions**)

Comment (zhongjiajie) [reply tests/hooks/test_hive_hook.py:479]: You right, a wrong remove

Comment (zhongjiajie) [reply tests/hooks/test_hive_hook.py]: You're right, remove wrong one.

**Comment by zhongjiajie** [comment]
CI green, PTAL @XD-DENG thanks.

**Comment by XD-DENG** [comment]
Thanks @zhongjiajie

**Comment by zhongjiajie** [comment]
Thanks @XD-DENG

**Pull request merged**